include/ringmesh/geomodel/builder/geomodel_builder_geology.h

```diff
@@ -76,8 +76,10 @@ namespace RINGMesh
             typename GeoModelGeologicalEntity< DIMENSION >::GEOL_FEATURE
                 geol_feature );
 
-        void set_mesh_entity_parent(
-            const gmme_id& child_gmme, index_t id, const gmge_id& parent_gmge );
+        void set_mesh_entity_parent( const gmme_id& child_gmme,
+            index_t id,
+            const gmge_id& parent_gmge,
+            bool update_parent = false );
```

Comment:
I see the interest of this boolean but is it used (true and false) in the code currently?

Reply:
The boolean as false is used in RINGMesh in the GeoModelBuilderRemoval. The boolean as true is used in my dependent library (in an executable used for creating and modifying geological features and geological entities).